clients/cli/src/error.rs

```diff
@@ -70,6 +70,7 @@ macro_rules! make_errkind_enum {
 }
 
 type IoError = std::io::Error;
+type PathBuf = std::path::PathBuf;
```

Comment:
I would just import `PathBuf` instead of type aliasing it.

Reply:
definitely